config/passportconfig.js

```diff
@@ -0,0 +1,50 @@
+var passport = require('passport');
+var localStrategy = require('passport-local').Strategy;
+const pool = require('../database/connection');
+
+
+passport.use('local', new localStrategy({ 
+    passReqToCallback : true
+  }, function(req, email, password, done) {
+          console.log('called local strategy');
+          console.log(email + " " + password)
+          var findUserQuery = "select * from users where email = 'teojunjie@gmail.com'";
```

Comment:
Need to move this to our SQL script. 
Also need to remove the hardcode.

Reply:
Yes noted definitely, i was just testing it out and making sure it works.